src/Tests/SenseNet.Tests/Implementations/InMemoryDataProvider.cs

```diff
@@ -3032,6 +3033,92 @@ public LogEntriesRow Clone()
         }
         #endregion
 
+        public static void SetContentHandler(Database db, string contentTypeName, string handler)
```

Comment:
This feature should be implemented at the provider level so the related test cases can work with any other provider.